File: kits/java/simple/lux/IOConstants.java
package lux;

public enum IOConstants {
    DONE("D_DONE"), RESEARCH_POINTS("rp"), RESOURCES("r"), UNITS("u"), CITY("c"), CITY_TILES("ct"), ROADS("ccd");
    public String str;

    IOConstants(final String s) {
        this.str = s;
    }

    @Override
    public String toString() {
        return this.str;
    }
}

File: kits/kotlin/simple/bot/src/main/kotlin/luxaibot/lux/IOConstants.kt
package luxaibot.lux

enum class IOConstants(val str: String) {
    DONE("D_DONE"), RESEARCH_POINTS("rp"), RESOURCES("r"), UNITS("u"), CITY("c"), CITY_TILES("ct"), ROADS("ccd");

    override fun toString(): String {
        return str
    }
}
